src/languages/en.js

```diff
@@ -353,9 +354,9 @@ export default {
         settledExpensify: 'Paid',
         settledElsewhere: 'Paid elsewhere',
         settledPaypalMe: 'Paid using Paypal.me',
-        settleExpensify: 'Pay with Expensify',
+        settleExpensify: ({formattedAmount}) => `Pay ${formattedAmount} with Expensify`,
         payElsewhere: 'Pay elsewhere',
```

Comment:
Here, the `formattedAmount` was not added to `payElsewhere` which was fixed in #34620